client/asset/dcr/dcr.go

```diff
@@ -377,8 +381,47 @@ type fundingCoin struct {
 // Driver implements asset.Driver.
 type Driver struct{}
 
-// Setup creates the DCR exchange wallet. Start the wallet with its Run method.
-func (d *Driver) Setup(cfg *asset.WalletConfig, logger dex.Logger, network dex.Network) (asset.Wallet, error) {
+// Exists checks the existence of the wallet. For the RPC wallet, this attempts
+// to connect and request the version to verify existence.
+func (d *Driver) Exists(walletType, dataDir string, settings map[string]string, net dex.Network) (bool, error) {
+	if walletType != walletTypeDcrwRPC {
+		return false, fmt.Errorf("no Decred wallet type %q available", walletType)
+	}
+	cfg, _, err := loadConfig(settings, net)
+	if err != nil {
+		return false, err
+	}
+	cl, err := rpcclient.New(&rpcclient.ConnConfig{
+		HTTPPostMode: true,
+		DisableTLS:   true,
+		Host:         cfg.RPCListen + "/wallet/" + cfg.Account,
```

Comment:
You're right about the host. It's not actually used anywhere, since we only check existence of seeded wallets. I should probably just define the method that way and get rid of this.